cmd/configuration/configuration.go

```diff
@@ -25,28 +25,28 @@ import (
 
 // Long descriptions are whitespace sensitive when generating docs using Sphinx.
 const (
-	initCmdShort = `Generates FlyteCTL config file in the user's home directory.`
-	initCmdLong  = `Creates a FlyteCTL config file in Flyte directory i.e ~/.flyte
+	initCmdShort = `Generate FlyteCTL config file in the user's home directory.`
+	initCmdLong  = `Create a FlyteCTL config file in Flyte directory i.e ~/.flyte
```

Comment:
Should this be "Generate" or "Generates"? IMO, "generates" sounds better. Not sure what is preferred though. What do you think, @SmritiSatyanV?